doc/.storybook/webpack.config.js

```diff
@@ -37,8 +37,10 @@ module.exports = ({ config, mode }) => {
   config.resolve.alias = {
     'react-native': 'react-native-web',
   };
+  
 
-  return withUnimodules(config, {
+  const wUni = withUnimodules(config, 
```

Comment:
Please change this to readable word. withUni?

Reply:
I fixed configWithExpo instead of wUni